windows-driver-docs-pr/install/inf-addservice-directive.md

```diff
@@ -224,6 +228,36 @@ This directive is almost never used in a *service-install-section*, but it might
 
 **BitReg**=*bit-registry-section*[**,**_bit-registry-section_]...  
 Is valid in a *service-install-section* but almost never used.
+ 
+**BootFlags**=*value*
```

Comment:
The section "Promoting a Driver's StartType at Boot Depending on Boot Scenario" later on this page should be updated to indicate that before Windows 11 version <TBD>, you should add BootFlags with the AddReg as it specifies but after Windows 11 version <TBD>, you should use this new BootFlags directive.